pkg/replication/replication.go

```diff
@@ -39,7 +39,7 @@ func (r Replicator) CheckPreparers() error {
 // wish to lock
 func (r Replicator) LockHosts(lock kp.Lock, overrideLock bool) error {
 	for _, host := range r.Nodes {
-		lockPath := kp.LockPath(host, r.Manifest.ID())
+		lockPath := kp.LockPath(kp.IntentPath(host, r.Manifest.ID()))
```

Comment:
In practice I very much doubt it's a problem. So far, it looks to me like lock conflicts in prod usually come from a deploy getting stuck or cancelled. The actual case of two people independently, unknowingly kicking off a deploy for the same pod and same node (which is where locking would save them from each other) seems quite rare.

It's also worth mentioning that replication locks are only held for the duration of the replication, so it's not like we have long-lived sessions in prod that would make the path change difficult. I don't think it'll end up being an issue.